surf/apps/communities/views.py

```diff
@@ -75,7 +66,10 @@ def collections(self, request, pk=None, **kwargs):
             serializer.is_valid(raise_exception=True)
             data = serializer.initial_data
             collection_ids = [d["id"] for d in data]
-            self._check_access(request.user, instance=instance, collection_ids=collection_ids)
+            try:
+                check_access_to_community(request.user, instance)
+            except AuthenticationFailed as exc:
+                return Response(str(exc), status=401)
```

Comment:
This is a bit nitty gritty, but normally an anonymous user would get 401 (unauthenticated) and a authenticated user that has no business with that particular community would get a 403 (forbidden). The first case may get a retry from the frontend, while in the second case the frontend should steer away